### ISSUE TITLE
mail server

### ISSUE THREAD

**Issue by Ghenghis** [xssc/gimme-ptc#3]
might need new mail server, seems like it isn't working anymore.

**Comment by xssc** [comment]
I will look into adding a new one today

**Comment by apazzy** [comment]
Not sure how complicated it would be to use GMail, but perhaps you can take advantage of the account+whatever@gmail.com stuff that this uses: https://github.com/FrostTheFox/ptc-acc-gen

**Comment by xssc** [comment]
@apazzy this is easily detectable and will be one of the first things fixed by PTC, not a good option

**Comment by thiemok** [comment]
Wouldn't using another throw away mail domain be as easy to detect as changing gmail suffixes on different accounts?

**Comment by xssc** [comment]
Yes, but I didn't change email from original. I am looking into better options

**Comment by thiemok** [comment]
Any leads on what that would be? I mean Undetectable mail accounts aren't that easy to come by.

**Comment by MrStu82** [comment]
an option? https://www.guerrillamail.com/GuerrillaMailAPI.html returns a json object with mail

**Comment by xssc** [comment]
Mail server is back up, but with a different domain, will be adding shortly

**Comment by denniskai1234** [comment]
@xssc any eta?

**Comment by xssc** [comment]
Well I can get just the email up in No time. But now Niantic made it so you have to accept TOS in game, so need to add that to

**Comment by denniskai1234** [comment]
@xssc are you able to do that?

**Comment by DrPiggy** [comment]
@xssc the tos is fine but is the mail fixed?

**Comment by xssc** [comment]
@DrPiggy it should be but PTC is down. I will upload the exe once I can confirm it, so PTC needs to be up

**Comment by DrPiggy** [comment]
@xssc okay btw i tried to run it myself with ruby but it seems to output:
`C:/Ruby200/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- nokogiri (LoadError)`

**Comment by xssc** [comment]
Make sure you run

`gem install bundle`

And then

`bundle install`

**Comment by DrPiggy** [comment]
@xssc well https://go.jooas.com/ says ptc is online but i tried too and it doesnt work

**Comment by denniskai1234** [comment]
@DrPiggy current version isnt published + he is speaking about the login page

**Comment by DrPiggy** [comment]
@denniskai1234 i used the uncompiled version denniskai

**Comment by DrPiggy** [comment]
@denniskai1234 also the login page works fine too

**Comment by denniskai1234** [comment]
@DrPiggy 

https://club.pokemon.com/us/pokemon-trainer-club/sign-up/

Not up, and yes the uncompiled version is not updated also.

**Comment by DrPiggy** [comment]
@denniskai1234 it is updated it was updated 1 hour ago and i thought you were talking about login so your half right ;p.

**Comment by DrPiggy** [comment]
@xssc its backup from maintance :)

**Comment by xssc** [comment]
It seems temp mail was blocked. Looking for other options

**Comment by DrPiggy** [comment]
@xssc yea found out that too

**Comment by DrPiggy** [comment]
@xssc maybe use https://www.guerrillamail.com/GuerrillaMailAPI.html and parse the json using the json module?

**Comment by xssc** [comment]
@DrPiggy will look into

**Comment by swesensei** [comment]
@DrPiggy  Tried that one but it won't let me chose the domain for the address in the api and the one that you get is blocked.

**Comment by ekimpadd** [comment]
@xssc check [accept-tos.py](https://github.com/scottstamp/pgoapi/blob/master/scripts/accept-tos.py) for accepting the in game TOS. So we only need to fix the e-mail server now :)

**Comment by xssc** [comment]
@ekimpadd That is python. I will be making my own ruby solution for this. I will makea script to activate already made accounts, and then also active new accounts when they are made

**Comment by ekimpadd** [comment]
@xssc I know it's pyton. This was more a heads up, because it uses the python "pgoapi" and therefore it is much easier to access the in-game TOS. But looking forward to your ruby solution :+1:

**Comment by jackfang888** [comment]
@xssc Not sure if it is related, but the generator has a 100% failure rate for me.  Just cloned it and ran (Fedora).  Are any logs generated that I could look through?

**Comment by DrPiggy** [comment]
@xssc firstly work on the mail so it will even create accounts and why would you want it to automatically accept tos in-game? players cant do that?

**Comment by knobunc** [comment]
To accept the tos, people can use https://github.com/scottstamp/pgoapi/blob/master/scripts/accept-tos.py

**Comment by xssc** [comment]
This project is in Ruby and it's goal is to make ptc accounts for pogo. So it will accept the ToS in game, and it will do so in Ruby. I am currently working on this and hope to push an update tomorrow

**Comment by darklordor** [comment]
so, we only have to wait until you upload the new version?

**Comment by xssc** [comment]
Yes, I'm still working on it, hopefully today.

**Comment by Aaranaw** [comment]
Hi xssc, have you had any luck with this?
Thanks.

**Comment by xssc** [comment]
Update: recaptcha has been added.

**Comment by Aaranaw** [comment]
Not seeing any new commits?

**Comment by xssc** [comment]
I should clarify: PTC added reCaptcha. This entire project will not work as is. Needs to be rebuilt from the ground up to work in the browser

**Comment by Aaranaw** [comment]
Oh,  thanks for the explanation.
